cbench/cbench.c

```diff
@@ -70,107 +74,154 @@ double run_test(int n_fakeswitches, struct fakeswitch * fakeswitches, int mstest
         timersub(&now, &then, &diff);
         if( (1000* diff.tv_sec  + (float)diff.tv_usec/1000)> total_wait)
             break;
-        for(i = 0; i< n_fakeswitches; i++)
-            fakeswitch_set_pollfd(&fakeswitches[i], &pollfds[i]);
 
-        poll(pollfds, n_fakeswitches, 1000);      // block until something is ready or 100ms passes
+        #ifdef USE_EPOLL
```

Comment:
please make indent consistent in cbench.c and fakeswitch.c

(use "1.$s/[press tab]/[press 4 spaces]/g" in vi)